db/Dockerfile

```diff
@@ -9,11 +9,12 @@ RUN apt-get update \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-ADD make_db.sh /docker-entrypoint-initdb.d/
-ADD setup-wale.sh /docker-entrypoint-initdb.d/
-COPY docker-entrypoint1.sh /
-RUN chmod +x /docker-entrypoint1.sh
```

Comment:
I wasn't aware of that!
I still think it is unecessary :) I think this repo should aim at publishing this image official, and people can docker pull.
It is an edge case, and it is just one line of code :) Feel free to do as you feel :)